clients/python/llmengine/model.py

```diff
@@ -72,41 +73,43 @@ def create(
                 Source of the LLM. Currently only HuggingFace is supported
 
             inference_framework (`LLMInferenceFramework`):
-                Inference framework for the LLM. Currently only DeepSpeed is supported
+                Inference framework for the LLM. Currently supported frameworks are `deepspeed`, `text_generation_inference`, `vllm`, and `lightllm`
 
             num_shards (`int`):
                 Number of shards for the LLM. When bigger than 1, LLM will be sharded
                 to multiple GPUs. Number of GPUs must be equal or larger than num_shards.
-                Only affects behavior for text-generation-inference models
 
             quantize (`Optional[Quantization]`):
-                Quantization for the LLM. Only affects behavior for text-generation-inference models
+                Quantization method for the LLM. `text_generation_inference` supports `bitsandbytes` and `vllm` supports `awq`.
 
             checkpoint_path (`Optional[str]`):
-                Path to the checkpoint for the LLM. For now we only support loading a tar file from AWS S3.
-                Safetensors are preferred but PyTorch checkpoints are also accepted (model loading will be slower).
-                Only affects behavior for text-generation-inference models
+                AWS S3 Path to the checkpoint for the LLM. LLM engine must have permission to access the given path.
```

Comment:
I think we can say "Remote path" instead of "AWS S3 Path", since this should technically be OK as long as LLM Engine can access the file. And restricting it to S3 is not what we want for self-hosted users.